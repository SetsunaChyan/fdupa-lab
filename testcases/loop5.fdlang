function main(){
x = 100;

while (i <= 10) {
    y = input();
    if (y < 128) {
        x = x - 1;
    } else {
        x = x + 1;
    }
    i = i + 1;
}

check_interval(x, 89, 111);
check_interval(x, 90, 110);
check_interval(x, 91, 109);
}