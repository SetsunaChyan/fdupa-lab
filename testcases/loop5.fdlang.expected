Line 14: YES; x in [89, 89] U [91, 91] U [93, 93] U [95, 95] U [97, 97] U [99, 99] U [101, 101] U [103, 103] U [105, 105] U [107, 107] U [109, 109] U [111, 111]
Line 15:  NO; x in [89, 89] U [91, 91] U [93, 93] U [95, 95] U [97, 97] U [99, 99] U [101, 101] U [103, 103] U [105, 105] U [107, 107] U [109, 109] U [111, 111]
Line 16:  NO; x in [89, 89] U [91, 91] U [93, 93] U [95, 95] U [97, 97] U [99, 99] U [101, 101] U [103, 103] U [105, 105] U [107, 107] U [109, 109] U [111, 111]
